scripts/update_bazel_workspace.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+# This command should be run from the emsdk root.
+
+function cleanup {
+  popd > /dev/null
+}
+
+ERR=0
+# Attempt to change to the emsdk root directory
+pushd "${PWD%%/emsdk*}/emsdk" > /dev/null 2> /dev/null
```

Comment:
If think you change this to `cd $(dirname $0)/..` it will mean you can run it from anywhere (and also avoid pushd)?